docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+---
+
+![Picture of the plant watering robot on a desk.](../../img/plant-watering-pi/plant-watering-robot.png)
+
+## Getting Started
+
+Building a useful robot doesn't have to require complicated code or expensive equipment.
+With a Raspberry Pi and some cheap, basic hardware, you can keep your plants healthy and happy from anywhere in the world!
+
+Follow this tutorial to learn how to set up an automatic plant watering system, as pictured above.
+You can leave this plant watering robot on your windowsill or desk, refill the water supply as needed, and easily program the system to your specifications for watering your plants.
+
+Complete the physical assembly and wiring, [create and connect to a new robot](#configure-the-components-of-your-robot-in-the-viam-app), [configure your robot's components](#configure-the-components-of-your-robot-in-the-viam-app), [configure a custom Capacitive Soil Moisture model](#configure-the-capacitive-soil-moisture-sensor-as-a-custom-sensor-component-model) [sensor](/components/sensor) component, and [code a Python file utilizing the Viam Python SDK](#add-python-control-code) to control the plant watering robot with `viam-server` and your Pi.
+
+Completing this tutorial requires the following hardware:
+
+- A Raspberry Pi with SD card
+- A [Capacitive Soil Moisture Sensor](https://www.amazon.com/KeeYees-Sensitivity-Moisture-Watering-Manager/dp/B07QXZC8TQ/ref=asc_df_B07QXZC8TQ/?tag=hyprod-20&linkCode=df0&hvadid=343238573411&hvpos=&hvnetw=g&hvrand=14606440922488452520&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-757549749596&psc=1&tag=&ref=&adgrpid=71762478951&hvpone=&hvptwo=&hvadid=343238573411&hvpos=&hvnetw=g&hvrand=14606440922488452520&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-757549749596)
+- A [Peristaltic Pump](https://www.amazon.com/Gikfun-Peristaltic-Connector-Aquarium-Analytic/dp/B01IUVHB8E/ref=asc_df_B01IUVHB8E/?tag=hyprod-20&linkCode=df0&hvadid=198093101467&hvpos=&hvnetw=g&hvrand=13835398343702336934&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-384674250225&psc=1) motor and [tubing](https://www.amazon.com/dp/B08H1ZD5VZ?psc=1&ref=ppx_yo2ov_dt_b_product_details)
+- An [Adafruit MCP3008 ADC](https://www.amazon.com/dp/B00NAY3RB2?psc=1&ref=ppx_yo2ov_dt_b_product_details)
+- A [Motor Speed Controller](https://www.amazon.com/CHENBO-Trigger-Adjustment-Electronic-Controller/dp/B099RF72R1/ref=asc_df_B099RF72R1/?tag=hyprod-20&linkCode=df0&hvadid=532384528241&hvpos=&hvnetw=g&hvrand=11376239784428845641&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9067609&hvtargid=pla-1410298730875&th=1)
+- A Breadboard
+- 9V Battery
+- Plant Box
+- Water Cup or Box
+
+Before starting this tutorial, follow the [Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to prepare your Pi to run `viam-server`.
+Make sure your Pi is flashed with a Viam-compatible operating system, and that you are able to SSH into it.
```

Comment:
Since you instruct them to do this in the configuration section you can remove it here
```suggestion
```

Reply:
This is actually a different guide than I am directing them to below-- would not want them to have to reflash after doing the SPI enablement/testing, but they don't need to create a new robot/install until after assembly and testing is done